conf/gunicorn-dev.conf.py

```diff
@@ -0,0 +1,5 @@
+bind = "0.0.0.0:9099"
+reload = True
+reload_extra_files = "checkmate/templates"
+timeout = 0
+workers = 2
```

Comment:
For some reason Checkmate had `workers = 4` in `conf/gunicorn/dev.conf.py`. I don't see any reason why it should be `4` rather than `2` so I'm just setting it to `2`, same as I did in LMS.